backend/src/main/resources/graphql/main.graphqls

```diff
@@ -456,4 +457,6 @@ type Mutation {
   ): String @requiredPermissions(allOf: ["UPDATE_TEST"])
   sendPatientLinkSms(internalId: String!): String
     @requiredPermissions(allOf: ["UPDATE_TEST"])
+  updateOrganization(type: String!): String
```

Comment:
**thought (non-blocking):** Unless I'm reading this wrong, I think this is a breaking change for an out of date UI

That said, I think it's a desirable one bc we don't want non-admins to be able to edit their own org name anymore

👍